elements/map/main.ts

```diff
@@ -208,6 +212,57 @@ export class EOxMap extends LitElement {
     return this._config;
   }
 
+  private _projection: ProjectionLike;
+
+  /**
+   * @type ProjectionLike
+   */
+  get projection() {
+    return this._projection;
+  }
+
+  /**
+   * projection of the map view as SRS-identifier (e.g. EPSG:4326)
+   */
+  @property({ attribute: "projection", type: String })
+  set projection(projection: ProjectionLike) {
+    const oldView = this.map.getView();
+    if (projection && projection !== oldView.getProjection().getCode()) {
+      const newCenter = transform(
```

Comment:
I didn't manage to successfully test this. When changing the projection from e.g. 3857 to 4326, then the zoom is reset to initial. I tried in storybook:

```
      <button
        @click=${() => {
          const eoxMap = document.querySelector("eox-map");
          eoxMap.setAttribute("projection", "EPSG:4326");
        }}
      >
        change
      </button>
```

Initial center: `center: [16.8, 48.2],`

Reply:
i couldn't reproduce this behaviour with zoom, but i did find weird interactions with the initial center, which came down with weird interactions and timings when setting the center of the view and setting the target of the map... this was quite tough to find, i restructured the `center` to be in line with the other attributes (with a getter and setter instead of watching for changes in the `updated`-hook), and setting the center once after the map was added to the `DOM`.

Also added an entry to the story book